transmart-core-api/src/main/groovy/org/transmartproject/core/users/AuthorisationChecks.groovy

```diff
@@ -0,0 +1,45 @@
+package org.transmartproject.core.users
+
+import org.transmartproject.core.ontology.OntologyTerm
+import org.transmartproject.core.ontology.Study
+import org.transmartproject.core.querytool.QueryDefinition
+import org.transmartproject.core.querytool.QueryResult
+
+interface AuthorisationChecks {
```

Comment:
Please name this `LegacyAuthorisationChecks` to signal that this is for the legacy model (based on study tree nodes).